src/posts.fr-CA.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: layouts/posts.njk
+title: Articles
+lang: fr-CA
+translationKey: posts
+eleventyNavigation:
+  parent: fr-CA
+  key: Articles
+  order: 3
+permalink: "/fr/articles/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}/{% endif %}"
```

Comment:
is the `fr` in the path going to be an issue if "fr-CA" is used in the URLs instead?

Reply:
It would be, but if the site builder was adding another French locale they would be creating another posts file (e.g. `posts.fr-FR.md`) and would be setting the permalinks in each.

Reply:
I agree that the fix for a site build is easy, but I worry that the debugging of this issue might be hard. Is there a way to surface this/draw attention to it. Even if it's just a note in the README or something.

Reply:
In 7309c82, I changed the permalink so it uses the `config.languages[lang].slug` value instead of a hard-coded `fr`. This should better guidance for adding other languages.